Doc/faq/design.rst

```diff
@@ -259,9 +259,8 @@ Why isn't there a switch or case statement in Python?
 -----------------------------------------------------
 
 You can do this easily enough with a sequence of ``if... elif... elif... else``.
-There have been some proposals for switch statement syntax, but there is no
-consensus (yet) on whether and how to do range tests.  See :pep:`275` for
-complete details and the current status.
+For literal values, or constants within a namespace, you can also use a
+``match ... case`` statement.
```

Comment:
I expect it would be better to rewrite or remove this section, but not now. IMO "match" *is* the missing switch/case statement.

Reply:
Or update the question to be: Is there a switch or case statement in Python?